pkg/retry/retryer.go

```diff
@@ -220,9 +220,6 @@ func (rc *Container) WithRetryIfNeeded(ctx context.Context, callOptRetry *Policy
 
 // NewRetryer build a retryer with policy
 func NewRetryer(p Policy, r *ShouldResultRetry, cbC *cbContainer) (retryer Retryer, err error) {
-	if !p.Enable {
-		return nil, nil
-	}
```

Comment:
This will change the behavior of Kitex.

For example, currently, if a `Policy{Enable: false}` is set to method `Echo`, Kitex will discard the policy, and when sending a request, Kitex will fallback to the wildcard policy.

With this change, Kitex will actually set the disabled policy to method `Echo` and will not fallback.

Reply:
Guess it's more reasonable than current behavior, because by setting a disabled policy, user are more likely to not fallback to wildcard policy.